superglue/lib/reducers/index.ts

```diff
@@ -195,10 +217,14 @@ export function pageReducer(state = {}, action) {
   }
 }
 
-export function metaReducer(state = {}, action) {
+export function superglueReducer(
```

Comment:
This removes the Meta reducer in favor of just the superglueReducer. It was an alias anyway. Small changes.

```diff
 exports.handleGraft = handleGraft;
-exports.metaReducer = metaReducer;
 exports.pageReducer = pageReducer;
```